frontend/src/screens/onboarding/Success.tsx

```diff
@@ -45,7 +45,7 @@ export function Success() {
         .
       </p>
 
-      <Link to="/" className="flex justify-center mt-8">
+      <Link to="/wallet" className="flex justify-center mt-8">
```

Comment:
```suggestion
      <Link to="/home" className="flex justify-center mt-8">
```

Reason: the user hasn't completed the onboarding yet and the checklist is on the home page

Reply:
done